src/Exercism.Analyzers.CSharp/Analyzers/WeighingMachine/WeighingMachineSyntaxFactory.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Linq;
+
+using Exercism.Analyzers.CSharp.Syntax;
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace Exercism.Analyzers.CSharp.Analyzers.WeighingMachine
+{
+    public static class WeighingMachineSyntaxFactory
+    {
+        public static MemberAccessExpressionSyntax GetMethodCalled(this SyntaxNode syntaxNode, string methodName) =>
```

Comment:
This looks like it could be useful for more than just this analyzer. Maybe `SyntaxNodeExtensions` is a better location?